pygac/tests/test_reader.py

```diff
@@ -174,29 +174,62 @@ def test__validate_header(self):
             head = {'data_set_name': b'\xea\xf8'}
             self.reader._validate_header(head)
 
-    def test__correct_data_set_name(self):
+    def test__correct_data_set_name_ebcdic_encoded_header_invalid_path(self):
```

Comment:
Can this method be replaced by parametrizing `test__correct_data_set_name_invalid_header_and_valid_file` with two encodings?

Reply:
Ooh, it's unittest. Nevermind then